src/main.rs

```diff
@@ -481,15 +486,11 @@ async fn start_ldk() {
 		} else {
 			// We're starting a fresh node.
 			restarting_node = false;
-			let getinfo_resp = bitcoind_client.get_blockchain_info().await;
-
-			let chain_params = ChainParameters {
-				network: args.network,
-				best_block: BestBlock::new(
-					getinfo_resp.latest_blockhash,
-					getinfo_resp.latest_height as u32,
-				),
-			};
+
+			let polled_best_block = polled_chain_tip.to_best_block();
```

Comment:
Since this uses a new API, we'll need to wait for the next release. We could form the `BestBlock` manually from the `ValidatedBlockHeader` in the interim, though.